common/lib/xmodule/xmodule/js/spec/.gitignore

```diff
@@ -7,3 +7,4 @@
 !time_spec.js
 !collapsible_spec.js
 !xmodule_spec.js
+!main_requirejs.js
```

Comment:
I don't know about this too but without this Git was ignoring this file for me. I will double check this.